.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: go
 
 go:
-- "1.11.x"
-- "1.12.x"
+- "1.15.x"
```

Comment:
Can we also add "1.16.x" to this? Its now the most recent version of go

Reply:
Good call. Bumped to 1.16.x